frappe/custom/doctype/customize_form/customize_form.json

```diff
@@ -23,6 +23,7 @@
   "image_view",
```

Comment:
Update `modified` of this file... or else changes will not be replicated while migration.